lib/utils.js

```diff
@@ -96,6 +96,15 @@ function getCredsForDTAsUPS() {
     return vcap['user-provided'].find(b => b.name.match(/dynatrace/)).credentials
 }
 
+function getCredsForCLSAsUPS() {
+  if (!process.env.VCAP_SERVICES) return
+  const vcap = JSON.parse(process.env.VCAP_SERVICES)
+
+  // to support connection via user-provided services, APMs requirement is that the instance name contains "cloud-logging"
```

Comment:
```suggestion
  // to support connection via user-provided services, the instance name must contain "cloud-logging"
```
not an APM requirement